core/liquidity/v2/sla.go

```diff
@@ -123,7 +154,11 @@ func (e *Engine) getMidPrice() (*num.Uint, error) {
 	return midPrice, nil
 }
 
-func (e *Engine) doesLPMeetsCommitment(party string) bool {
+func (e *Engine) doesLPMeetsCommitment(
+	party string,
+	markPrice *num.Uint,
+	positionFactor num.Decimal,
+) bool {
```

Comment:
re `one.Sub(e.priceRange)` and `one.Add(e.priceRange)` consider keeping these instead of the net param to avoid recalculating every block.